### ISSUE TITLE
[Wordpress] Welcome to Nginx page

### ISSUE THREAD

**Issue by connorpekovic** [bitnami/vms#339]
### Platform

AWS

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

Unable to download Bitnami Diagnostic Tool

### bndiagnostic was not useful. Could you please tell us why?

Unable to download Bitnami Diagnostoc Tool on AWS Lightsail Instance

### Describe your issue as much as you can

Hi Everbody,

I have been using Bitnami Wordpress on AWS Lightsail for a year now.  When I go the home page (www.connorpek.com), now it now says "Welcome to nginx!".  I didn't change any configuration files in the terminal, and my wordpress content seems to be in the right file locations.

My Wordpress installation lives at the same file path as everybody else with Bitnami wordpress, which is `home/bitnami/stack/wordpress`.  Research indicated my server isn't pointing my home page to the right file, which should be `home/bitnami/stack/wordpress/index.php` like everyone elses. Research also indicates I could make that change in the `.htaccess` file, but I am unable to locate that file in my Lightsail Bitnami Wordpress installation.  Research also indicates the PHP logs would give me more detail, but I'm unauthorized to access files in`home/bitnami/stack/php/logs`.

Thanks a bunch,
Connor

**Comment by jotamartos** [comment]
Hi @connorpekovic,

I can't confirm the issue if you do not provide the Diagnostic Tool information but I think the issue is that you installed a NGINX server using the system packages and that one is preventing the Bitnami's Apache to run. Let's confirm it

```
sudo /opt/bitnami/ctlscript.sh status
ps aux | grep -i nginx
```

This should show that Apache is not running and that there is a NGINX process running in the instance. Is that correct? Let's stop it now 

```
sudo service nginx stop
sudo service nginx disable
sudo /opt/bitnami/ctlscript.sh start apache
```

Can you access your site now?

**Comment by connorpekovic** [comment]
Hi @jotamartos 

Accepted Answer

Hi Gabriel,

Thanks Yes somehow I downloaded Nginx and I had to disable it then boom, the website worked again. I disabled Nginx by using the following command before rebooting the aws lightsail server.

```
sudo update-rc.d -f nginx disable
```

After rebooting, Nginx doesnt show up on the list with either of the following commands:

```
sudo /opt/bitnami/ctlscript.sh status
ps aux | grep -i nginx

```

```
systemctl list-units
```

At this point the website is working. This problem was solved in part with assistance of AWS support [over on their support forum re:Post](https://repost.aws/questions/QUIM0E5dacR5WHPVI_kAihdQ/lightsail-wordpress-welcome-to-nginx).

Thanks a bunch, Connor
